integration_tests/cucumber.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  default: "",
-  critical: "--format @cucumber/pretty-formatter --tags @critical",
-};
+  default: '',
+  critical: '--format @cucumber/pretty-formatter --tags @critical'
+}
```

Comment:
I think there's a conflict between prettier and eslint here (single quotes vs double quotes, and semi-colons)